packages/wasm/src/services/credential/pex-helpers.js

```diff
@@ -49,23 +60,37 @@ function getAttributeName({field, selectedCredentials, index}) {
   return attributeName;
 }
 
-export function pexToBounds(
-  pexRequest,
-  selectedCredentials = [],
-  isRawBoundies = false,
-) {
+export function pexToBounds(pexRequest, selectedCredentials = [], removeFromRequest = false) {
```

Comment:
API validation uses `removeFromRequest: true`, but i dont think that will effect the bounds. You shouldnt need to use it on the wallet side, for the API "mock wallet" we set to false